PCbuild/build.bat

```diff
@@ -175,6 +175,15 @@ echo on
  /p:UseTestMarker=%UseTestMarker% %GITProperty%^
  %1 %2 %3 %4 %5 %6 %7 %8 %9
 
+if "%platf%"=="Win32" set prefix=%dir%win32
+if "%platf%"=="x64" set prefix=%dir%amd64
+if "%platf%"=="ARM64" set prefix=%dir%arm64
+if "%platf%"=="ARM" set prefix=%dir%arm32
+set suffix=
+if "%conf%"=="Debug" set suffix=_d
+set built_python_exe=%prefix%\python%suffix%.exe
+call "%built_python_exe%" -m ensurepip.bundle -v
```

Comment:
There's a handy `"%dir%\..\python.bat"` that you can use instead.

Reply:
Is it guaranteed to locate the built interpreter version? I was a bit lost and haven't found how this file is generated to check it myself..

Reply:
I applied the suggested change. Marking this thread as resolved.

Reply:
@zware could you take a look at https://dev.azure.com/Python/cpython/_build/results?buildId=136036&view=logs&j=91c152bd-7320-5194-b252-1404e56e2478&t=c7e99cd8-4756-5292-d34b-246ff5fc615f&l=764 — any idea why it might be failing? I don't see how it might be related to the PR, but OTOH, it doesn't crash on `main`…

Reply:
That particular build is apparently trying to run an `arm64` build on what is apparently an `amd64` runner, which isn't going to work out well :)